src/NHSD.BuyingCatalogue.Ordering.Services/DefaultDeliveryDateService.cs

```diff
@@ -21,30 +21,29 @@ public DefaultDeliveryDateService(ApplicationDbContext context)
 
         public async Task<DateTime?> GetDefaultDeliveryDate(CallOffId callOffId, CatalogueItemId catalogueItemId)
         {
-            Expression<Func<Order, IEnumerable<DefaultDeliveryDate>>> defaultDeliveryDate = o
+            Expression<Func<Order, IEnumerable<DefaultDeliveryDate>>> defaultDeliveryDateExpression = o
```

Comment:
While this obviously an expression I think I preferred the original name `defaultDeliveryDate` as it just seems to fit better in the EF Core query below.

Reply:
Reverted expression to name `defaultDeliveryDate `